devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp

```diff
@@ -115,14 +118,32 @@ YarpRobotLoggerDevice::YarpRobotLoggerDevice()
     // the logging message are streamed using yarp
     BipedalLocomotion::TextLogging::LoggerBuilder::setFactory(
         std::make_shared<BipedalLocomotion::TextLogging::YarpLoggerFactory>());
+
+    m_sendDataRT = false;
 }
 
 YarpRobotLoggerDevice::~YarpRobotLoggerDevice() = default;
 
 bool YarpRobotLoggerDevice::open(yarp::os::Searchable& config)
 {
+
     constexpr auto logPrefix = "[YarpRobotLoggerDevice::open]";
     auto params = std::make_shared<ParametersHandler::YarpImplementation>(config);
+    std::string remote;
+    m_sendDataRT = params->getParameter("remote", remote);
```

Comment:
The ``remote`` name is a bit generic. Can you move it to a group? You could call the group ``REAL_TIME_STREAMING`` or similar. In this way, you can set ``m_sendDataRT`` to ``true`` when the group exists.

Reply:
Fixed in [4ea7f85](https://github.com/ami-iit/bipedal-locomotion-framework/commit/4ea7f85117a66453647b79b9aaf39ffcb1aa6cd5)